frontend/src/component/common/ApiTokenTable/useApiTokenTable.tsx

```diff
@@ -22,15 +27,16 @@ export const useApiTokenTable = (
         return [
             {
                 id: 'Icon',
-                width: '1%',
```

Comment:
Out of curiosity, How does `1%` actually work on the table? Because I don't think it is actually 1%, is it?

Reply:
On tables that are not virtualized (actual `display:table`, instead of flexbox) it will indicate that this column should be squeezed to content.

Reply:
Cool; thanks!